freeform_json_to_yaml.py

```diff
@@ -0,0 +1,392 @@
+import yaml, argparse, json
+from random import randint
+from logger import LogLevel, Logger
+
+LOGGER = Logger('JsonConverter')
+
+class Indenter(yaml.Dumper):
+    ### Source: https://stackoverflow.com/questions/25108581/python-yaml-dump-bad-indentation#:~:text=class%20MyDumper(yaml.Dumper)%3A%0A%0A%20%20%20%20def%20increase_indent(self%2C%20flow%3DFalse%2C%20indentless%3DFalse)%3A%0A%20%20%20%20%20%20%20%20return%20super(MyDumper%2C%20self).increase_indent(flow%2C%20False)
+    def increase_indent(self, flow=False, indentless=False):
+        return super(Indenter, self).increase_indent(flow, False)
+
+SUPPLY_MAP = {
+    "hemostaticGauzeCount": "Hemostatic gauze",
+    "tourniquetCount": "Tourniquet",
+    "pressureBandageCount": "Pressure bandage",
+    "decompressionNeedleCount": "Decompression Needle",
+    "nasopharyngealAirwayCount": "Nasopharyngeal airway",
+    "pulseOximeterAvailable": "Pulse Oximeter",
+    "painMedsCount": "Pain Medications",
+    "splintCount": "Splint",
+    "ivBloodCount": "Blood",
+    "ivSalineCount": "IV Bag",
+    "burnDressingCount": "Burn Dressing",
+    "epiPenCount": "Epi Pen",
+    "chestSealCount": "Vented Chest Seal",
+    "blanketCount": "Blanket",
+    "lollipopCount": "Fentanyl Lollipop"
+}
+
+MENTAL_STATUS_MAP = {
+    "agony": "AGONY",
+    "calm": "CALM",
+    "unresponsive": "UNRESPONSIVE",
+    "dead": "UNRESPONSIVE",
+    "upset": "UPSET"
+}
+
+BREATHING_MAP = {
+    "normal": "NORMAL",
+    "fast": "FAST",
+    "restricted": "RESTRICTED",
+    "none": "NONE",
+    "collapsedLeft": "RESTRICTED",
+    "collapsedRight": "RESTRICTED",
+    "collapsedBoth": "RESTRICTED"
+}
+
+PULSE_MAP = {
+    "none": "NONE",
+    "faint": "FAINT",
+    "normal": "NORMAL",
+    "fast": "FAST"
+}
+
+INJ_LOC_MAP = {
```

Comment:
This isn't handling injury types of "C Chest Puncture", "Asthmatic" or "Forehead Scrape".